Gemfile

```diff
@@ -32,7 +32,6 @@ group :development do
   gem 'listen', '>= 3.0.5', '< 3.2'
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
-  gem 'web-console', '>= 3.3.0'
```

Comment:
@awead this could have stayed though ;) guessing you will add it with the other PR.